### PR TITLE
Matako

### PR DESCRIPTION
**Pull request by matakevin**

### CODE REVIEW

File: src/main/java/com/nci/automation/web/JavascriptUtils.java
Comment (iamdez99) [inline src/main/java/com/nci/automation/web/JavascriptUtils.java:1]: Please delete line 4 and format file.

File: src/main/java/com/nci/automation/web/CommonUtils.java
Comment (iamdez99) [inline src/main/java/com/nci/automation/web/CommonUtils.java:1]: Please format file.

File: src/main/java/com/nci/automation/web/WebDriverUtils.java
Comment (iamdez99) [inline src/main/java/com/nci/automation/web/WebDriverUtils.java:1]: Please format file.

File: src/test/java/AnalysisTools/ACT24/Features/ParticipantPortal.feature
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Features/ParticipantPortal.feature:1]: Feature file is empty.

File: src/test/java/AnalysisTools/ACT24/Features/ResearcherPortal.feature
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Features/ResearcherPortal.feature:1]: Feature file is empty.

File: src/test/java/AnalysisTools/ACT24/Pages/ACT24ParticipantPortalPage.java
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Pages/ACT24ParticipantPortalPage.java:1]: Page has no Web Elements.

File: src/test/java/AnalysisTools/ACT24/Pages/ACT24ResearcherPortalPage.java
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Pages/ACT24ResearcherPortalPage.java:1]: Page has no Web Elements.

File: src/test/java/AnalysisTools/ACT24/Steps/ACT24ParticipantPortalSteps.java
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Steps/ACT24ParticipantPortalSteps.java:1]: Page has no Web Elements.

File: src/test/java/AnalysisTools/ACT24/Steps/ACT24ResearcherPortalSteps.java
Comment (iamdez99) [inline src/test/java/AnalysisTools/ACT24/Steps/ACT24ResearcherPortalSteps.java:1]: File is empty.

**Review by iamdez99 — CHANGES_REQUESTED** [review]

File: src/main/java/com/nci/automation/web/JavascriptUtils.java
Comment (matakevin) [reply src/main/java/com/nci/automation/web/JavascriptUtils.java:1]: Removed lines 3 and 4.  Line 3 wasn't being used so I deleted it too

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**